src/packet/types.ts

```diff
@@ -0,0 +1,67 @@
+// DISCV5 message packet types
+
+import { EthereumNodeRecord } from "../enr/enr";
+
+export type Tag = Buffer; // TAG_LENGTH
+export type Nonce = Buffer; // ID_NONCE_LENGTH
+export type AuthTag = Buffer; // AUTH_TAG_LENGTH
+export type Magic = Buffer; // MAGIC_LENGTH
```

Comment:
I would consider moving all types to single file somewhere so it's easier to navigate